### DISCUSSION TITLE
Docker compose

### DISCUSSION THREAD

**Discussion by Ronald-Agorsah** [wg-easy/wg-easy#900] [Ideas]
cant we edit the tutorial to work with an docker compose file so that it would be easyer to automate

**Comment by eandersons** [comment]
The project already contains Docker Compose file: [`docker-compose.yml`](https://github.com/wg-easy/wg-easy/blob/master/docker-compose.yml).

I just created #901 to add installation and update instructions for Docker Compose as well.

**Reply by Ronald-Agorsah** [reply]
Thats great  maybe make sure to include the update process via Docker Compose aswell?

**Reply by eandersons** [reply]
> I just created https://github.com/wg-easy/wg-easy/pull/901 to add installation and **update** instructions for Docker Compose as well.
